composer.json

```
@@ -33,6 +33,7 @@
         "symfony/http-kernel": "^3.4|^4.2",
         "symfony/inflector": "^3.4|^4.2",
         "symfony/monolog-bridge": "^3.4|^4.2",
+        "symfony/mailer": "^3.4|^4.2",
```

```suggestion
        "symfony/mailer": "^4.4",
```

The mailer was introduced in symfony 4.3, but as experimental. So require the first "stable" version